packages/shared/renderer/sagas.ts

```diff
@@ -85,7 +87,11 @@ function* reportRealmChangeToRenderer() {
       const contentServerUrl: string = yield select(getAllowedContentServer, configuredContentServer)
       const current = convertCurrentRealmType(realmAdapter, contentServerUrl)
       defaultLogger.info('UpdateRealmsInfo', current)
-      getUnityInstance().UpdateRealmsInfo({ current })
+      if (getFeatureFlagEnabled(store.getState(), 'realm_about_enabled')) {
+        getUnityInstance().UpdateRealmsInfo({ current })
+      } else {
+        getUnityInstance().UpdateRealmAbout(realmAdapter.about)
+      }
```

Comment:
This works like a charm even though the function may not exist. I'd keep both for maximum compatibility
```suggestion
      getUnityInstance().UpdateRealmsInfo({ current })
      getUnityInstance().UpdateRealmAbout(realmAdapter.about)
```